backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -96,11 +98,13 @@ public record CreateProjectResponse(Guid? Id, CreateProjectResult Result);
     {
         permissionService.AssertCanManageProjectMemberRole(input.ProjectId, input.UserId);
         var projectUser =
-            await dbContext.ProjectUsers.Include(r => r.User).FirstOrDefaultAsync(u =>
+            await dbContext.ProjectUsers.Include(r => r.Project).Include(r => r.User).FirstOrDefaultAsync(u =>
                 u.ProjectId == input.ProjectId && u.UserId == input.UserId);
         if (projectUser is null) throw new NotFoundException("Project member not found");
         projectUser.Role = input.Role;
         projectUser.User.UpdateCreateProjectsPermission(input.Role);
+        projectUser.User.UpdateUpdatedDate();
+        projectUser.Project.UpdateUpdatedDate();
```

Comment:
```suggestion
        projectUser.Project.UpdateUpdatedDate();
        projectUser.UpdateUpdatedDate()
```
project user also has an updated date, should be updating it here too

Reply:
Done in commit 2f9eaf78.